packages/vitepress-plugin-og-image-generator/src/index.ts

```diff
@@ -0,0 +1,97 @@
+import { fileURLToPath } from 'node:url'
+import { dirname, relative, resolve, sep } from 'node:path'
+import fs from 'fs-extra'
+import fg from 'fast-glob'
+import { Resvg } from '@resvg/resvg-js'
+import type { ArticleTree, Doc } from './types/metadata'
+import { removeEmoji } from './utils'
+
+const DIR_ROOT = resolve(dirname(fileURLToPath(import.meta.url)), '..')
```

Comment:
What happens when this is one of the module that installed under `node_modules` when specifying relative path like this? What would `DIR_ROOT` be?